pkg/test/controller/k8s.go

```diff
@@ -81,9 +81,9 @@ func assertEventRecorded(t *testing.T, c client.Client, kind, name, namespace, r
 func assertEventNotRecorded(t *testing.T, c client.Client, kind, name, namespace, reason string) {
 	err := waitUntilEventRecorded(t, c, kind, name, namespace, reason)
 	if err == nil {
-		t.Errorf("expected event with reason '%v' to not be recorded for %v %v/%v, but it was", reason, kind, namespace, name)
+		t.Errorf("Error: expected event with reason '%v' to not be recorded for %v %v/%v, but it was", reason, kind, namespace, name)
```

Comment:
Nit: It's okay to leave it as is, though just want to point out that in our code base, most error strings (if not all) are short sentences or phrases that don't have a indicator in the beginning.

Reply:
I don't quite get why we want to add the prefix "Error" and the golinter shall complain that it starts with a capital letter.  

@gemmahou Do you mind sharing the debugging log you have and you expect this change to improve regarding "our dynamic test just fails without a clear error message"?